deps/chakrashim/include/v8.h

```diff
@@ -734,9 +734,9 @@ class V8_EXPORT HandleScope {
  private:
   friend class EscapableHandleScope;
   template <class T> friend class Local;
-  static const int kOnStackLocals = 5;  // Arbitrary number of refs on stack
+  static const int kOnStackLocals = 8;  // Arbitrary number of refs on stack
 
-  JsValueRef _locals[kOnStackLocals];   // Save some refs on stack
+  JsValueRef _locals[kOnStackLocals + 1];   // Save some refs on stack
   JsValueRef _refs;                     // More refs go to a JS array
```

Comment:
last nit: `_refs` field is now unneeded. Use _locals[0] directly. (Remember to init _locals or _locals[0] at least in constructor).